bokeh/models/layouts.py

```diff
@@ -281,6 +279,15 @@ def __init__(self, *args, **kwargs) -> None:
     CSS pseudo selector to access the root DOM element.
     """)
 
+    context_menu = Nullable(Instance(Menu), default=None, help="""
+    The menu to display when user right clicks on the component.
+
+    .. note::
+        To display the native context menu, the user needs to right
+        click twice. The second click closes the user-defined menu
+        and falls back to the native one.
```

Comment:
```suggestion
        To display a native context menu, the viewer will need to right-click
        twice. The second click closes the Bokeh context menu and falls back
        back the native one.
```

cc @tcmetzger I usually prefer "a" instead of "the" in docs like this, but curious if you have any thoughts. 

@mattpap I don't recall ever encountering an interaction pattern like this. Was there an existing system that inspired this UX? Is there some way we can signal to viewers to make things more discoverable?

Reply:
> I don't recall ever encountering an interaction pattern like this. Was there an existing system that inspired this UX? Is there some way we can signal to viewers to make things more discoverable?

YouTube is the most prominent example of this particular behavior. JLab, for example, the moves native menu under `shift+contextmenu`, with information about this in the customized menu:
![image](https://user-images.githubusercontent.com/27475/173257115-3c5b512e-101c-48a7-a130-a818aba8e91b.png)
They also stopped overriding `ctrl+shift+i` (devtools), so looks like the UI there has gotten some more consideration.

Reply:
> They also stopped overriding ctrl+shift+i (devtools), so looks like the UI there has gotten some more consideration.

Actually no, they still get this wrong when a notebook is open.